src/menu.ts

```diff
@@ -236,7 +236,7 @@ function getAppMenu(
 					label: __( 'Studio Help' ),
 					click: async () => {
 						const locale = await getUserLocaleWithFallback();
-						void shellOpenExternalWrapper( getDocsLink( locale, 'studio' ) );
+						void shellOpenExternalWrapper( translateLink( locale, DOCS_LINKS.studio ) );
```

Comment:
I think it was clearer when function accepted the link name as a parameter.

Reply:
👍 changed!